### DISCUSSION TITLE
Merci pour cet outil que je trouve très bon et que j'apprécie !

### DISCUSSION THREAD

**Discussion by LauwCost** [senshu/Sozi#629] [Show and tell]
Bonjour à toutes et tous, 

Désolé, pour le français mais mon anglais est trop mauvais.

Je voulais remercier tous les contributeurs et toutes les contributrices du projet car je trouve cet outil extrêmement intéressant et il me convient bien.
Du coup, j'ai fait un (petit) don :).

J'aurais 2 questions que je vais poster par ailleurs mais je souhaitais partager la réalisation faites grâce au couple inkscape / Sozi :
C'est [ici](https://ecn.cemea.asso.fr/communs-numeriques/#diapo-accueil) (et c'est en CC-BY-SA, n'hésitez pas :).

Librement, 
LauwCost

**Comment by aumouvantsillage** [comment]
Merci pour le témoignage et pour le don :)
C'est toujours agréable de voir que de nouveaux utilisateurs s'approprient cet outil et qu'il peut inspirer des créations intéressantes.
